Update integration-management with note about partner ints (#22042)
DataDog/documentation: content/en/agent/guide/integration-management.md

The Agent comes with a set of bundled official Datadog integrations to allow users to start monitoring their applications quickly. These integrations are available as single Python packages, and you can upgrade them separately.

> **Note:** Non-core integrations such as those built by partners or community members are not retained when the Agent is upgraded. Please be mindful that these integrations need to be re-installed upon upgrading the Agent version.

Reviewer: ```suggestion
**Note**: Community and Marketplace integrations are not retained when the Agent is upgraded. These integrations need to be re-installed upon upgrading the Agent version.
```

Author: @cswatt what do you think of **Note**: Community, Partner and Marketplace integrations are not retained when the Agent is upgraded. These integrations need to be re-installed upon upgrading the Agent version.
